apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -983,19 +979,21 @@ export class WorkshopForm extends React.Component {
                 <HelpBlock>{validation.help.location_name}</HelpBlock>
               </FormGroup>
             </Col>
-            <Col sm={6}>
-              <FormGroup>
-                <ControlLabel>Location Address (optional)</ControlLabel>
-                <MapboxLocationSearchField
-                  placeholder="Enter a location"
-                  onChange={this.handleLocationChange}
-                  value={this.state.location_address || ''}
-                  readOnly={this.props.readOnly}
-                  style={this.getInputStyle()}
-                  className={'form-control'}
-                />
-              </FormGroup>
-            </Col>
+            {!this.state.virtual && (
```

Comment:
Yeah, from Tess:
`This is our regional partners using a field in a way it wasnt designed.  Theres about 200 already scheduled workshops so we probably won't have success if we need partners to go through and change this before summer workshops.   We should talk about the right way to do this before Annual Year Workshops start getting scheduled, but for now we are in a bit of limbo.
`

Reply:
Tess and I talked about a couple of options, and decided to use the thing the partners were doing already for now, but I think it'd make more sense as a separate field in the long term!

Reply:
That makes sense -- I'm curious if we're worried about how this will read for virtual workshops that have already been created, with location names that aren't links? Like this one?

https://studio.code.org/pd/workshop_dashboard/workshops/10273

Reply:
oh yeah, good point, it looks like they're sending emails themselves, and are probably just putting the link to join there. 
we could parse the location_name and check for a url, which would add some complexity but make it more durable for any case, happy to do that
@tess323 any thoughts?